teams/codebender.md

```diff
@@ -35,13 +36,21 @@ permalink: /teams/codebender/
  * Write-up on my participation in Insomni'hack 2016: ([Blog Post on the Smartcat Challenge](http://cs3281-2.blogspot.sg/2016/01/te-insomnihack-2016-teaser-smartcat-web.html))
 
 #### @bsmmoon (Moon Seokmin)
-* ...
+* Jan 19 - Received project details
```

Comment:
I prefer to see `Week 2 - Wed` format so that it is easy to determine at which point of the semester something was done.